### PR TITLE
New slots component to allow progressive upgrades

### PR DESCRIPTION
**Pull request by sglanzer**
#major#

# CHANGELOG

Modified the interface of the object browser to minimize coupling between the presentation and logic layers.

See https://github.com/ciena-frost/ember-frost-object-browser#api for the new interface specification.

A new `frost-object-browser-inline` component has also been introduced that implements the `frost-object-browser` component according to the previously stable 5.x interface and marked the usage of this component as deprecated for removal in the next major release (15.x).  This allows users of the legacy format a smoother transition path to the new slots based implementation.

See https://github.com/ciena-frost/ember-frost-object-browser#inline-api for the legacy inline specification.

### CODE REVIEW

**Comment by sglanzer** [comment]
I'm going to be offline for the next few hours, but I'll check back later today for comments and to complete the PR

**Comment by sglanzer** [comment]
An overview of the changes starts here in the README https://github.com/ciena-frost/ember-frost-object-browser/pull/51/files#diff-04c6e90faac2675aa89e2176d2eec7d8R25

The rest of the PR may be of less interest

Tests and a new `frost-list` component with controls are next

File: README.md [hunk 1]
```diff
@@ -1,120 +1,117 @@
...
+| ` ` | ` ` | ` ` | Coming soon |
 
 ## Examples
-
-### Template
-
+### Template:
 ```handlebars
-{{#frost-object-browser
+{f#rost-object-browser
```
Comment (TheAxeMan301) [inline README.md]: typo

**Comment by cstolli** [comment]
👍 can we merge this?  I'd like to opt-out of block-slots and used the old interface for now.

**Comment by TheAxeMan301** [comment]
It's all right with me. Probably too late to argue but I would prefer to see the deprecated old syntax moved to a frost-object-browser-no-slots component so that frost-object-browser will always be the main one. But whatever.

**Comment by sglanzer** [comment]
@TheAxeMan301 I could do that, I just wanted to leave this as a no-risk upgrade for people with the 5.x implementation

@cstolli @job13er @sandersky Any opinions on that?

**Comment by sandersky** [comment]
@sglanzer I've already made the upgrade to the latest so it would be nice if I don't have to change anything on my end to upgrade again. But I really don't have a strong opinion either way.

**Comment by sglanzer** [comment]
Ok, I'm going to shift to @TheAxeMan301's suggestion prior to merge then and use `frost-object-browser` and `frost-object-browser-inline` as the two components

**Pull request merged**